CONTRIBUTING.md

```diff
@@ -200,6 +200,18 @@ to run only the test you just created for this step.
 
 For more information, see the `docs for mpl-test <https://github.com/astrofrog/pytest-mpl>`_.
 
+
+## Cached Data Files
+MetPy keeps some test data, as well as things like shape files for US counties in a data cahce
+supported by the pooch library. To add files to this, please ensure they are the minimum size.
```

Comment:
I would add that this command should be run in the same directory that contains the staticdata directory - just to make it clear.